worker/tasks/views.py

```diff
@@ -0,0 +1,25 @@
+from typing import Tuple, List
+
+from celery import shared_task
+
+from tasks.tables import ViewTable, TableData, TableInfo
+
+
+@shared_task
+def get_views() -> List[TableInfo]:
+    pass
+
+
+@shared_task
+def create_view(view: ViewTable) -> List[TableInfo]:
+    pass
+
+
+@shared_task
+def get_view(view_name: str) -> Tuple[TableInfo, TableData]:
```

Comment:
Discuss with Simos:
A view is created on the main hospital data. We should never be able to see the data of a view.

Reply:
We will throw an exception if the view is private.